src/scmdata/netcdf.py

```diff
@@ -54,254 +30,250 @@
 
 
 def _var_to_nc(var):
-    return var.replace("|", "__").replace(" ", "_")
```

Comment:
I would lean towards keeping the old variable name replacements and adding an attribute to each variable with the original variable name. Perhaps `long_name`? that is what the long variable names are labelled as in CF compliant nc files

Reply:
That's a better solution, done.